annotator/static/app.js

```diff
@@ -12,5 +12,7 @@ $(() => {
         $container: $("#player"),
         videoSrc: window.video.location,
         videoId: window.video.id,
+        videoStart: window.video.start_time,
```

Comment:
Like, instead of deconstructing the `video` object, the Player class can deconstruct it. Like we call `new Player(..., video)` instead of `new Player(..., video.location, video.id, video.start_time etc)`